src/main/java/com/mraof/minestuck/item/MSItems.java

```diff
@@ -267,6 +322,7 @@
 	public static final RegistryObject<Item> EDISONS_SERENITY = REGISTER.register("edisons_serenity", () -> new WeaponItem(new WeaponItem.Builder(Tiers.IRON, 7, -2.6F).efficiency(1.0F).set(MSItemTypes.MISC_TOOL).set(ItemRightClickEffect.switchTo(MSItems.EDISONS_FURY)), new Item.Properties()));
 	
 	public static final RegistryObject<Item> SKAIA_FORK = REGISTER.register("skaia_fork", () -> new WeaponItem(new WeaponItem.Builder(MSItemTypes.REGI_TIER, 9, -2.6F).efficiency(1.0F).set(MSItemTypes.MISC_TOOL), new Item.Properties().tab(MSItemGroup.WEAPONS).rarity(Rarity.RARE)));
+	public static final RegistryObject<Item> SKAIAN_CROCKER_ROCKER = REGISTER.register("skaian_crocker_rocker", () -> new WeaponItem(new WeaponItem.Builder(MSItemTypes.REGI_TIER, 10, -2.6F).efficiency(1.0F).set(MSItemTypes.MISC_TOOL), new Item.Properties().defaultDurability(2048).tab(MSItemGroup.WEAPONS).rarity(Rarity.RARE)));
```

Comment:
With a damage recalc value of 16.9, this would be the most power weapon in the game before random damage or other factors are considered. I think I could be fine with this, but we should discuss what that means

Reply:
generally as we add m0re enemies, and with the current w0rk 0n the rebalancing 0f the existing 0nes thr0ugh unique abilities, it seems fitting players sh0uld be able t0 create m0re p0werful weap0ns. 
i wasnt aware it w0uld be the str0ngest th0ugh t0 be h0nest! i just figured it deserved an extra attack p0int since its a c0mbinati0n 0f tw0 pretty p0werful f0rks, and making it just the same damage value as the skaia f0rk is kind 0f lame :p

Reply:
That is fair! It would probably be best to increase the damage provided by the Welsh and Zilly tier weapons but that can occur in another PR at a later date